ci_framework/roles/edpm_build_images/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Install edpm-image-builder
+  ansible.builtin.include_tasks: install.yml
+
+- name: Find latest image name
+  import_role:
+    name: discover_latest_image
+
+- name: Set needed facts
+  set_fact:
+    cacheable: true
```

Comment:
You do not need to include the cacheable attribute in the set_fact task if you want to reuse the variable within the same playbook. The cacheable attribute is used to control whether the fact should be cached for future playbook runs or not.